src/pages/Beta/Pool/ClaimReward/index.tsx

```diff
@@ -46,20 +46,17 @@ const ClaimReward = ({ stakingInfo, version, onClose }: ClaimProps) => {
       const method = version < 2 ? 'getReward' : 'harvest'
       const args = version < 2 ? [] : [poolMap[stakingInfo.stakedAmount.token.address], account]
 
-      await stakingContract[method](...args)
-        .then((response: TransactionResponse) => {
-          addTransaction(response, {
-            summary: t('earn.claimAccumulated', { symbol: 'PNG' })
-          })
-          setHash(response.hash)
-        })
-        .catch((error: any) => {
-          setAttempting(false)
-          // we only care if the error is something _other_ than the user rejected the tx
-          if (error?.code !== 4001) {
-            console.error(error)
-          }
```

Comment:
I removed because typescript accuses error because the typing of "error" is unknown
![image](https://user-images.githubusercontent.com/37405304/175096462-2027fd57-d0b7-4a12-8673-b6c443d64947.png)

Reply:
that we can fix by adding `"useUnknownInCatchVariables": false` to tsconfig file. but that condition is important incase user reject tx in manually